api/src/main/java/org/pmiops/workbench/api/InstitutionController.java

```diff
@@ -117,7 +121,10 @@ public ResponseEntity<Institution> updateInstitution(
     institutionService
         .getAffiliatedUsers(shortName)
         .forEach(
-            u -> accessSyncService.updateUserAccessTiers(u, Agent.asAdmin(userProvider.get())));
+            u ->
+                accessSyncService.updateUserAccessTiers(
+                    u, Agent.asAdmin(userProvider.get()), false));
```

Comment:
For what it's worth, I think this linter-enforced style is terrible. I would:
```suggestion
            u -> accessSyncService
                .updateUserAccessTiers(u, Agent.asAdmin(userProvider.get()), false));
```